frontend/lib/src/hooks/useWidgetManagerElementState.tsx

```diff
@@ -61,6 +64,20 @@ const useWidgetManagerElementState = <T,>({
     [widgetMgr, id, key]
   )
 
+  useEffect(() => {
```

Comment:
**suggestion:** This should leverage the existing `useFormClearHelper` for this logic rather than reimplementing it

Reply:
I originally had it doing that, but forgot the reason I couldn't use that... let me look again

Reply:
Ah yes, it comes down to some typescript types issues with formId looking like it's required (and a string) using that helper where as the `useWidgetManagerElementState` allows formId to (correctly?) be undefined.

Reply:
I guess I could make the `formId` optional in `FormElementProtoInterface`?`

Reply:
I think I'd leave the `FormElementProtoInterface` alone since it accurately represents the proto backend. Perhaps we could utilize `formId: formId || ''` on the FE since this is going through a valid form id check anyway

Reply:
Hmmmm, semantically though, why is it not optional? Not every widget is in a form? Or are you making the point that since that's what it is (even if it isn't semantically correct, we should not differ from the source of truth (proto).

Reply:
The latter!

Reply:
sounds good